app/presenters/statistics_presenter.rb

```diff
@@ -95,28 +93,10 @@ def local_comments
                                .count
   end
 
-  def all_services_helper
-    result = {}
-    Configuration::KNOWN_SERVICES.each {|service, options|
-      result[service.to_s] = AppConfig.show_service?(service, nil)
-    }
-    result
-  end
-
-  def all_services
-    @all_services ||= all_services_helper
-  end
-
   def available_services
     Configuration::KNOWN_SERVICES.select {|service|
       AppConfig.show_service?(service, nil)
```

Comment:
I don't get the test at https://github.com/diaspora/diaspora/blob/develop/lib/configuration_methods.rb#L41 with "authorized" etc. We just need to know if the service is enabled or not.